nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/src/main/java/org/apache/nifi/processors/azure/eventhub/GetAzureEventHub.java

```diff
@@ -198,16 +201,16 @@
         return propertyDescriptors;
     }
 
-
-    protected void setupReceiver(final String connectionString) throws ProcessException {
+    protected void setupReceiver(final String connectionString, final ScheduledExecutorService executor) throws ProcessException {
         try {
-            eventHubClient = EventHubClient.createFromConnectionString(connectionString).get();
-        } catch (InterruptedException | ExecutionException | IOException | ServiceBusException e) {
+            EventHubClientImpl.USER_AGENT = "ApacheNifi-azureeventhub/2.3.2";
```

Comment:
@mysunnytime ApacheNiFi (not the capitalized F).  I realize that seems unimportant but either it should be all lowercase or it should follow convention which is NiFi (not Nifi).  So either do 'apachenifi-azureeventhub/2.3.2" or do 'ApacheNiFi-azureeventhub/2.3.2' . I prefer the NiFi convention but either is fine.

Reply:
Cool! Thanks for pointing out. I'll fix that.